packages/app-store/locations.ts

```diff
@@ -314,6 +316,47 @@ export const getHumanReadableLocationValue = (
   return linkValue || "";
 };
 
+export const getLocationHtml = (event: CalendarEvent) => {
+  const providerName = guessEventLocationType(event.location)?.label;
+
+  const location = event.location;
+  let meetingUrl = location?.search(/^https?:/) !== -1 ? location : undefined;
+
+  if (event) {
+    meetingUrl = getVideoCallUrlFromCalEvent(event) || meetingUrl;
+  }
+
+  const isPhone = location?.startsWith("+");
+
+  if (meetingUrl) {
+    return `
+      <a
+        href=${meetingUrl}
+        target="_blank"
+        title="Meeting URL"
+        style="color: '#101010'"
+        rel="noreferrer"
+      >
+        ${providerName || "Link"}
```

Comment:
This is not the right place to declare it. Also, this is similar to our LocationInfo function. Why did you introduce a new function? Aren't you using it?